aptos-move/framework/cached-packages/src/aptos_sdk_builder.rs

```diff
@@ -1,2489 +0,0 @@
-// Copyright (c) Aptos
```

Comment:
Please regenerate.

Reply:
I don't think this file is auto-generated and should be checked in. 

we already have `aptos_framework_sdk_builder.rs` generated from aptos-framework package
we also have `aptos_token_sdk_builder.rs` generated from aptos-token package.